scripts/configpair.sh

```
@@ -35,8 +35,8 @@ setup_ns() {
     NSIM_DEV_2_NAME=$(find $NSIM_DEV_2_SYS/net -maxdepth 1 -type d ! \
         -path $NSIM_DEV_2_SYS/net -exec basename {} \;)
 
-    PODMAN_NODE1_NS=$(get_podman_netns_id $CONTAINER1)
-    PODMAN_NODE2_NS=$(get_podman_netns_id $CONTAINER2)
+    PODMAN_NODE1_NS=$(get_container_netns_id $CONTAINER1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)